bcipy/helpers/tests/test_load.py

```diff
@@ -27,7 +27,8 @@
         "fields": [
             {
                 "age": {
-                    "required": "false"
+                    "required": "false",
+                    'anonymize': 'true'
```

Comment:
We should be consistent in our use of single and double quotes. I believe the JSON format typically uses double quotes.